### PR TITLE
IAP-3839 Update Jan21 Release Notes for APIP

### PR DESCRIPTION
**Pull request by dginevaAxway**
https://deploy-preview-1553--axway-open-docs.netlify.app/docs/apim_relnotes/20210130_apip_relnotes/

Automatically generated by Netlify CMS

### CODE REVIEW

File: content/en/docs/apim_relnotes/20210130_apip_relnotes.md [hunk 1]
```diff
@@ -15,7 +14,10 @@ API Portal is available as a software installation or a virtualized deployment i
 
 * If you are installing API Portal for the first time using this update, see [Install API Portal](/docs/apim_installation/apiportal_install/)
 * If you are already using API Portal (7.5.x, 7.6.x, 7.7.x) and want to install this update, see [Upgrade API Portal](/docs/apim_installation/apiportal_install/upgrade_automatic/)
-* If you want to deploy API Portal in Docker containers, see [Deploy API Portal in containers](/docs/apim_installation/apiportal_docker/)
+* If you want to deploy API Portal in Docker containers, see [Deploy API Portal in containers](/docs/apim_installation/apiportal_docker/docker_portal_upgrade/)
+* If you are already using API Portal in Docker containers and you want to install this update, see 
+
+  [Update API Portal in Docker containers](/docs/apim_installation/apiportal_docker/)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210130_apip_relnotes.md:20]: How come the page 'Deploy API Portal in containers' is useful to customers who are already using API Portal and want to install this update?

File: content/en/docs/apim_relnotes/20210130_apip_relnotes.md [hunk 2]
```diff
@@ -15,7 +14,10 @@ API Portal is available as a software installation or a virtualized deployment i
 
 * If you are installing API Portal for the first time using this update, see [Install API Portal](/docs/apim_installation/apiportal_install/)
 * If you are already using API Portal (7.5.x, 7.6.x, 7.7.x) and want to install this update, see [Upgrade API Portal](/docs/apim_installation/apiportal_install/upgrade_automatic/)
-* If you want to deploy API Portal in Docker containers, see [Deploy API Portal in containers](/docs/apim_installation/apiportal_docker/)
+* If you want to deploy API Portal in Docker containers, see [Deploy API Portal in containers](/docs/apim_installation/apiportal_docker/docker_portal_upgrade/)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210130_apip_relnotes.md:17]: Customers who want to deploy API Portal in Docker containers should read 'Upgrade API Portal container deployment' page ??

**Comment by jowens-axway** [comment]
Content looks ok to me, still a few placeholders to complete, will review it again closer to completion

File: content/en/docs/apim_relnotes/20210130_apip_relnotes.md (hunk 1 above)
Comment (dginevaAxway) [reply content/en/docs/apim_relnotes/20210130_apip_relnotes.md:20]: this is the same as the second point in the same question but for docker. If you have already installed API portal on docker this reference is how to upgrade it in docker.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**